src/pico/i_video.c

```diff
@@ -103,12 +104,12 @@ static const patch_t *stbar;
 
 volatile uint8_t interp_in_use;
 
-#define USE_1280x1024x60 1
-
 // display has been set up?
 
 static boolean initialized = false;
 
+static uint32_t frame = 0;
```

Comment:
this was mostly for a race condition I was chasing that ended up being a multithreading issue. it's still nice to have, but I might remove it in the future.